backend/graphql/resolvers/testSessionResolvers.ts

```diff
@@ -0,0 +1,70 @@
+import { GraphQLScalarType, Kind } from "graphql";
+import TestSessionService from "../../services/implementations/testSessionService";
+import UserService from "../../services/implementations/userService";
+import TestService from "../../services/implementations/testService";
+import SchoolService from "../../services/implementations/schoolService";
+
+import {
+  ITestSessionService,
+  TestSessionRequestDTO,
+  TestSessionResponseDTO,
+} from "../../services/interfaces/testSessionService";
+import { ITestService } from "../../services/interfaces/testService";
+import IUserService from "../../services/interfaces/userService";
+import { ISchoolService } from "../../services/interfaces/schoolService";
+
+const userService: IUserService = new UserService();
+const schoolService: ISchoolService = new SchoolService(userService);
+const testService: ITestService = new TestService();
+const testSessionService: ITestSessionService = new TestSessionService(
+  testService,
+  userService,
+  schoolService,
+);
+
+const testSessionResolvers = {
+  NumberOrArrayOrNull: new GraphQLScalarType({
+    name: "NumberOrArrayOrNull",
+    description: "A Number or An Array or Null",
+    serialize(value) {
+      if (
+        typeof value === "number" ||
+        typeof value === null ||
+        (Array.isArray(value) &&
+          value.every(function (e) {
+            return typeof e === "number";
+          }))
+      ) {
+        return value;
+      }
+      throw new Error("Answers must be either a Number or Array or Null");
+    },
+    // parseValue and parseLiteral will need to be created for creating test sessions mutation
```

Comment:
what does this mean?

Reply:
Was referring to the "answers" field but yeah that's way too vague lol I will rewrite it

Reply:
oh i meant what does your comment mean "// parseValue and parseLiteral will need to be created for creating test sessions mutation"

Reply:
ohhh for createTestSession, the NumberOrArrayOrNull type under results field will need the parseValue and parseLiteral functions (created kind of similarly to the serialize function I created) in order to parse the input to the correct type and to check if it is the correct type. The serialize function is for "get" and parseValue/parseLiteral is for "create".